### DISCUSSION TITLE
Telemetry disabled in VS Code

### DISCUSSION THREAD

**Discussion by vacharbhargav008** [VSCodium/vscodium#1538] [Q&A] [answered]
`https://code.visualstudio.com/docs/supporting/faq` This link shows this image:

![](https://cdn.discordapp.com/attachments/1034537971052986368/1114665944451657728/image.png)

What is the difference that VSCodium provides if Microsoft allows you to turn off telemetry in VSCode?

**Comment by paulcarroty** [comment, marked as answer]
https://github.com/VSCodium/vscodium/discussions/1539#discussioncomment-6090032

- https://github.com/VSCodium/vscodium/blob/master/patches/cli.patch
- https://github.com/VSCodium/vscodium/blob/master/undo_telemetry.sh
- https://github.com/VSCodium/vscodium/tree/master/patches

**Reply by beedell-roke** [reply]
@paulcarroty, you've not linked to commits, so some of these are 404 as of now.

**Reply by paulcarroty** [reply]
Fixed.

**Reply by beedell-roke** [reply]
@paulcarroty, albeit solely until they're modified again. I meant:

1. [`blob/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/patches/cli.patch`](https://github.com/VSCodium/vscodium/blob/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/patches/cli.patch#L1C1-L311C52:~:text=vscodium/-,patches,/cli.patch,-Latest%20commit)

1. [`blob/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/undo_telemetry.sh`](https://github.com/VSCodium/vscodium/blob/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/undo_telemetry.sh#L1C1-L45C37:~:text=/-,undo_telemetry.sh,-Latest%20commit)

1. [`tree/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/patches`](https://github.com/VSCodium/vscodium/tree/d2cdc6e515b8fc0b62e24a5646474fd8171461b7/patches#:~:text=/-,patches,-/)
